### PR TITLE
Add an openstack image type

### PR DESCRIPTION
**Pull request by bcl**
This is a qcow2 image with cloud-init in the template.

### CODE REVIEW

**Comment by larskarlitski** [comment]
Ah, cool idea to split this out into a separate image type. Thanks!

@jgiardino If this is accepted, it's another one for https://github.com/weldr/welder-web/pull/373.

**Comment by stefwalter** [comment]
@jgiardino How about "OpenStack QCOW2 Image (.qcow2)"

File: share/composer/openstack.ks [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Lorax Composer openstack output kickstart template
+
+# Firewall configuration
+firewall --enabled
```
Comment (stefwalter) [inline share/composer/openstack.ks]: This should change to:

```
firewall --disabled
```

Openstack has its own firewall, and this gets in the way.

File: share/composer/openstack.ks [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+logging --level=info
+# Shutdown after installation
+shutdown
+# System timezone
+timezone  US/Eastern
+# System bootloader configuration
+bootloader --location=mbr
+
+# Start cloud-init at boot time
+services --enabled=cloud-init
```
Comment (stefwalter) [inline share/composer/openstack.ks]: I had to change this to: 

```
services --enabled=sshd,cloud-init,cloud-init-local,cloud-config,cloud-final
```

to get it to boot in Openstack

**Comment by stefwalter** [comment]
I needed this FIXUP to get it to boot in openstack and show a console there properly:

```
diff --git a/share/composer/openstack.ks b/share/composer/openstack.ks
index 2d0fdf72..a242fa38 100644
--- a/share/composer/openstack.ks
+++ b/share/composer/openstack.ks
@@ -1,7 +1,7 @@
 # Lorax Composer openstack output kickstart template
 
 # Firewall configuration
-firewall --enabled
+firewall --disabled
 
 # NOTE: The root account is locked by default
 # Network information
@@ -19,10 +19,10 @@ shutdown
 # System timezone
 timezone  US/Eastern
 # System bootloader configuration
-bootloader --location=mbr
+bootloader --location=mbr --append="no_timer_check console=ttyS0,115200n8 console=tty1 net.ifnames=0"
 
 # Start cloud-init at boot time
-services --enabled=cloud-init
+services --enabled=sshd,cloud-init,cloud-init-local,cloud-config,cloud-final
 
 %post
 # Remove random-seed
```

**Comment by stefwalter** [comment]
I can confirm that with these changes, I am able to:

 * Compose an image
 * Upload an image to Openstack
 * Boot an instance
 * Log into that instance

**Pull request closed without merging**